pkgs/development/libraries/qt-5/5.12/default.nix

```diff
@@ -79,6 +79,15 @@ let
         url = "https://git.archlinux.org/svntogit/packages.git/plain/trunk/qtbug-77037-workaround.patch?h=packages/qt5-webengine&id=fc77d6b3d5ec74e421b58f199efceb2593cbf951";
         sha256 = "1gv733qfdn9746nbqqxzyjx4ijjqkkb7zb71nxax49nna5bri3am";
       })
+      # patch for CVE-2019-13720, can be removed when it is included in the next upstream release
+      # https://bugreports.qt.io/browse/QTBUG-1019226
+      (fetchpatch {
+        chromium/third_party/blink/renderer/modules/webaudio/convolver_node.cc
+        name = "qtwebengine-CVE-2019-13720.patch";
+        url = "https://code.qt.io/cgit/qt/qtwebengine-chromium.git/patch/?id=d6e5fc10";
+        sha256 = "0fa4c9silihsawhnlaknbcg28rqp53dy79azsfy6qzgyic27m2b5";
+        postFetch = "sed 's;chromium/third_party/blink/renderer/modules/webaudio/convolver_node.cc;src/tird_party/&;' -i $out";
```

Comment:
Watch out: there is a typo in "tird_party"

Reply:
Thanks!